tests/linearizability/failpoints.go

```diff
@@ -33,7 +36,8 @@ const (
 )
 
 var (
-	KillFailpoint                            Failpoint = killFailpoint{}
+	KillFailpoint                            Failpoint = killFailpoint{target: AnyMember}
+	EnableAuthKillFailpoint                  Failpoint = killFailpoint{enableAuth: true, target: Follower}
```

Comment:
Do you need to target follower here? I understand that 1471 happens when follower is killed, however we don't need to hardcode it.